network/codec/cbor/codec.go

```diff
@@ -85,7 +85,7 @@ func (c *Codec) Encode(v interface{}) ([]byte, error) {
 // NOTE: 'envelope' contains 'code' & serialized / encoded 'v'.
 // i.e.  1st byte is 'code' and remaining bytes are CBOR encoded 'v'.
 // Expected error returns during normal operations:
-//   - codec.UnknownMsgCodeErr if message code byte does not match any of the configured message codes.
+//   - codec.ErrMsgUnmarshal if message code byte does not match any of the configured message codes.
 //   - codec.ErrMsgUnmarshal if the codec fails to unmarshal the data to the message type denoted by the message code.
 func (c *Codec) Decode(data []byte) (interface{}, error) {
```

Comment:
I think the binstat framework is not used anymore. We started removing code like the following from the implementations: 
* https://github.com/onflow/flow-go/blob/04b4bcaf206cac75e956a557a959668b100d9356/network/codec/cbor/codec.go#L91-L96
* https://github.com/onflow/flow-go/blob/04b4bcaf206cac75e956a557a959668b100d9356/network/codec/cbor/codec.go#L103
* https://github.com/onflow/flow-go/blob/04b4bcaf206cac75e956a557a959668b100d9356/network/codec/cbor/codec.go#L105
would you mind taking it out here as well?